### ISSUE TITLE
Questions about rendering on the DTU

### ISSUE THREAD

**Issue by zf-123** [city-super/GSDF#22]
This is a very interesting work, thanks to the authors for making it open source. When I use the configuration in the code, the rendering result of dtu is very different from the paper, can you give me some comments?

![Image](https://github.com/user-attachments/assets/e904cfb7-a163-4365-9772-534ffc4f3c31)

![Image](https://github.com/user-attachments/assets/bd7c80c5-cd67-44e1-90b3-f7d4fc0bc13e)

![Image](https://github.com/user-attachments/assets/25998136-a2f9-44d5-bf82-1b9b3724a564)

**Comment by Zerui-Yu** [comment]
The paper divides the training set and the test set, and reports the results of the test set, whereas all the images in your experiment are the training set, so each metric is high

**Comment by zf-123** [comment]
> The paper divides the training set and the test set, and reports the results of the test set, whereas all the images in your experiment are the training set, so each metric is high

Thanks for your answer, I made sure to divide the training set and test set. Unfortunately, I followed the config of the paper and did not get the results improved by the paper, getting metrics that are all far below what the paper provides.

![Image](https://github.com/user-attachments/assets/d19ae247-6418-465c-aa27-29cb801d242e)

**Comment by Zerui-Yu** [comment]
Oh sorry, I thought earlier that the second line was the metrics for the paper. It does look like the metrics in the paper are very high, but what strikes me as odd is that even if all the images are the training set, the results should not be this high for reporting only the training set. For example, the 2DGS paper has the following results (PSNR in the TRAINING-SET). 

![Image](https://github.com/user-attachments/assets/39875b80-0a7d-4b71-a808-8ecafa30e1df)

The PSNR for the training set is around 34, which is about the same as the 2DGS results I reproduced, and in the GSDF table it's 38.76, which is clearly not normal (Not to mention that GSDF even divided the test set in the paper).

**Comment by Zerui-Yu** [comment]
Oh sorry, I thought earlier that the second line was the metrics for the paper. It does look like the metrics in the paper are very high.  I'm not quite sure why this is the case, but other issues mention that the chamfer distance when reconstructing the mesh also differs significantly from the paper's.
